packages/drawer/src/react/__specs__/__snapshots__/storyshots.spec.tsx.snap

```diff
@@ -25,7 +25,7 @@ exports[`Storyshots drawer controlled: external state 1`] = `
     role="button"
   >
     <p
-      data-css-r3ja7m=""
+      data-css-1joltl8=""
```

Comment:
Can you speak to the reason that the snapshots in this pr have changed?  As I understand it now, they shouldn't have changed because no styles/values have changed, only the variables used to represent those values.